src/vivarium_nih_us_cvd/components/effects.py

```diff
@@ -0,0 +1,100 @@
+from typing import Callable
+
+import numpy as np
+import pandas as pd
+from vivarium.framework.engine import Builder
+from vivarium.framework.time import get_time_stamp
+
+from vivarium_nih_us_cvd.constants import data_values
+
+
+class OutreachEffect:
+    """A component to model the impact of the outreach risk on medication adherence levels"""
+
+    def __init__(self):
+        self.exposure_pipeline_name = data_values.PIPELINES.OUTREACH_EXPOSURE
+        self.sbp_target_pipeline_name = (
+            data_values.PIPELINES.SBP_MEDICATION_ADHERENCE_EXPOSURE
+        )
+        self.ldlc_target_pipeline_name = (
+            data_values.PIPELINES.LDLC_MEDICATION_ADHERENCE_EXPOSURE
+        )
+
+    def __repr__(self):
+        return "OutreachEffect"
+
+    ##############
+    # Properties #
+    ##############
+
+    @property
+    def name(self) -> str:
+        return f"outreach_effect"
+
+    #################
+    # Setup methods #
+    #################
+
+    def setup(self, builder: Builder) -> None:
+        self.randomness = builder.randomness.get_stream(self.name)
+        self.population_view = builder.population.get_view(["age", "sex"])
```

Comment:
Oh yeah, I wanted to ask about that; it's no longer used since I'm not modifying the 'outreach' column here, but I was originally needing to load the state table and initialize the 'outreach' column. However, that initialization did not rely on any other columns - but it seems like you can't get a view of just the index?